api/src/test/java/org/pmiops/workbench/api/UserControllerTest.java

```diff
@@ -404,13 +404,13 @@ public void listBillingAccounts_upgradeYES_freeYES_cloudYES() throws IOException
     assertThat(response.getBillingAccounts()).isEqualTo(expectedWorkbenchBillingAccounts);
   }
 
-  // billing upgrade is true, free tier is available, no cloud accounts
```

Comment:
"billing upgrade" is an old feature flag.  It's always true now.  It means that it's possible for users to provide their own billing.